sources/osgShadow/ShadowMapAtlas.js

```diff
@@ -379,32 +379,32 @@ utils.createPrototypeObject(
 
             for (var i = 0, l = this._shadowMaps.length; i < l; i++) {
                 var shadowMap = this._shadowMaps[i];
-                if (shadowMap.isEnabled() && !shadowMap.isFilledOnce()) {
+                if (shadowMap.isContinuousUpdate() || shadowMap.needRedraw()) {
                     shadowMap.cullShadowCasting(cullVisitor, bbox);
                 }
             }
         },
 
-        isEnabled: function() {
+        isContinuousUpdate: function() {
```

Comment:
Probably more correct getContinuousUpdate.

Reply:
Seems to me osgjs code is more in favor of is... (isDirty, isEnabled, etc)
Can change, but better do it consistently ? Whole bag of cat opening there :)

Reply:
@Kuranes I mean isContinuousUpdate seems weird in english, but I'm no native.

Reply:
ha well, just applied the pattern consistently instead of natively speaker
"it's a getter?" => "is" + "var name" + ()